topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -430,21 +428,159 @@ A powerful tool to visualize the content of BAM files is the Integrative Genomic
+> >    > > 3. From the percentage of X+Y reads, most of the reads map to X and only few to Y. So only few genes may be on Y. So the samples are probably both female.
+> >    > >
+> >    > >    ![Samtools idxstats](../../images/ref-based/samtools-idxstats-xy-plot.png)
+> >    > {: .solution}
+> >    {: .question}
+> {: .hands_on}
+>
+> #### Gene body coverage
+>
+> The gene body is the different regions of a gene. It is important to check if reads coverage is uniform over gene body or if there is any 5'/3' bias. For example, a bias towards the 5' end of genes could indicate degradation of the RNA. Alternatively, a 3' bias could indicate that the data is from a 3' assay. To assess this, we can use the **Gene Body Coverage** tool from the RSeQC ({% cite wang2012rseqc %}) tool suite. This tool scales all transcripts to 100 nucleotides (using a provided annotation file) and calculates the number of reads covering each nucleotide position.
+>
+> > ### {% icon hands_on %} Hands-on: Check gene body coverage
+> >
+> > 1. **Convert GTF to BED12** {% icon tool %} to convert the GTF file to BED:
+> >    - {% icon param-file %} *"GTF File to convert"*: `Drosophila_melanogaster.BDGP6.87.gtf`
+> >
+> > 2. **Gene Body Coverage (BAM)** {% icon tool %}:
+> >    - *"Run each sample separately, or combine mutiple samples into one plot"*: `Run each sample separately`
+> >      - {% icon param-files %} *"Input .bam file"*: `mapped.bam` files (outputs of **RNA STAR** {% icon tool %})
+> >    - *"Reference gene model"*: BED12 file (output **Convert GTF to BED12** {% icon tool %})
+> >
+> > 3. **MultiQC** {% icon tool %} to aggregate the idxstats logs:
+> >    - In *"Results"*
+> >      - *"Which tool was used generate logs?"*: `RSeQC`
+> >      - In *"RSeQC output"*
+> >         - *"Type of RSeQC output?"*: `gene_body_coverage`
+> >         - {% icon param-files %} *"RSeQC gene_body_coverage output"*: `(text)` files (outputs of **Gene Body Coverage (BAM)** {% icon tool %})
+> >
+> >    > ### {% icon question %} Question
+> >    >
+> >    > ![Gene body coverage](../../images/ref-based/rseqc_gene_body_coverage_plot.png)
+> >    >
+> >    > How are the coverage across gene bodies? Are the samples biased in 3' or 5'?
+> >    >
+> >    > > ### {% icon solution %} Solution
+> >    > >
+> >    > > For both samples there is a pretty even coverage from 5' to 3' ends (despite some noise in the middle). So no obvious bias in both samples.
+> >    > {: .solution}
+> >    {: .question}
+> {: .hands_on}
+>
+> #### Read distribution across features
+>
+> With RNA-Seq data, we expect most reads to map to exons rather than introns or intergenic regions. Before going further in counting and differential expression analysis, it may be interesting to check the distribution of reads across known gene features (exons, CDS, 5'UTR, 3'UTR, introns, intergenic regions). For example, a high number of reads mapping to intergenic regions may indicate the presence of DNA contamination.
```

Comment:
```suggestion
> With RNA-Seq data, we expect most reads to map to exons rather than introns or intergenic regions. Before going further in counting and differential expression analysis, it may be interesting to check the distribution of reads across known gene features (exons, CDS, 5' UTR, 3' UTR, introns, intergenic regions). For example, a high number of reads mapping to intergenic regions may indicate the presence of DNA contamination.
```